package.json

```diff
@@ -126,20 +120,24 @@
     "react-test-renderer": "^16.0.0",
     "rewrite-files": "^1.0.1",
     "rimraf": "^2.6.1",
+    "seek-style-guide-webpack": "^2.0.1",
     "semantic-release": "^6.3.2",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0",
     "static-site-generator-webpack-plugin": "^3.4.1",
-    "string-replace-loader": "^1.2.0",
+    "string-replace-loader": "^1.3.0",
     "stylelint": "8.0.0",
     "stylelint-config-css-modules": "^1.0.0",
     "stylelint-config-standard": "^16.0.0",
+    "style-loader": "0.19.0",
     "superstatic": "^4.0.3",
     "surge": "^0.18.0",
+    "svgo": "0.7.2",
+    "svgo-loader": "1.2.1",
     "validate-commit-msg": "^2.8.2",
-    "webpack": "^1.13.1",
-    "webpack-dev-server": "^1.14.1",
-    "webpack-fail-plugin": "^1.0.6",
+    "webpack": "^3.6.0",
+    "webpack-dev-server": "^2.9.1",
+    "webpack-fail-plugin": "^2.0.0",
```

Comment:
The fail plugin I believe was only required in webpack 1, where it didn't have the correct exit code if there was an error.

Reply:
Nice catch 👍